server/src/main/resources/db/changelog/20180430101420-add-intent-fields-to-consumer.xml

```diff
@@ -7,14 +7,61 @@
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet id="20180430101420-1" author="vrjain">
-        <comment> add-intent-fields-to-consumer</comment>
+        <comment> Add system purpose role column to cp_consumer</comment>
         <addColumn tableName="cp_consumer">
-            <column name="offering" type="VARCHAR(255)"/>
+            <column name="sp_role" type="VARCHAR(255)"/>
```

Comment:
Just making sure, but this changeSet hasn't been release anywhere, right?

Reply:
Correct. That's why I reused the existing instead of creating yet another.